lightning/src/util/config.rs

```diff
@@ -206,7 +206,26 @@ pub struct ChannelConfig {
 	/// This cannot be changed after a channel has been initialized.
 	///
 	/// Default value: true.
-	pub commit_upfront_shutdown_pubkey: bool
+	pub commit_upfront_shutdown_pubkey: bool,
+	/// When we close a channel cooperatively with our counterparty, we negotiate a fee for the
+	/// closing transaction which both sides find acceptable, ultimately paid by the channel
+	/// funder/initiator.
+	///
+	/// When we are the funder, because we have to pay the channel closing fee, we bound the
+	/// acceptable fee by our [`Background`] and [`Normal`] fees, with the upper bound increased by
+	/// this value. Because the on-chain fee we'd pay to force-close the channel is kept near our
+	/// [`Normal`] feerate during normal operation, this value represents the additional fee we're
+	/// willing to pay in order to avoid waiting for our counterparty's to_self_delay to reclaim our
+	/// funds.
+	///
+	/// When we are not the funder, we require the closing transaction fee pay at least our
+	/// [`Background`] fee estimate, but allow our counterparty to pay as much fee as they like.
+	///
+	/// Default value: 1000 satoshis.
```

Comment:
Well a smarter value setting could be based on `to_self_delay` length and channel value size? Hmmm...

Reply:
Sure, there's lots of things we could do, but adding more knobs does not better enable users to control things, usually the opposite, knobs are just confusing. If users want to do really fine-grained control based on the channel details they can set the target when they call close (or update this value, when we implement that).